### DISCUSSION TITLE
Feature Request: Undo for the tree

### DISCUSSION THREAD

**Discussion by Ddmmiittrryy** [TriliumNext/Trilium#1443] [Ideas]
Hello
It would be very useful to have an option to undo accidental move of nodes in a complex tree.
It could be done either in standard Ctrl+Z command or in a separate command.

Thanks

**Comment by zadam** [comment]
Hi, yes, I understand the need. Unfortunately it's not a simple feature and there are higher priority things to do. Maybe someday!

**Reply by zDEFz** [reply]
Have done this before. Undo/Redo is ridiculously complex and probably needs an overhaul of the whole software to function properly.
